vivaldi/hooks/window-panel-tab-switched-actions.js

```diff
@@ -0,0 +1,86 @@
+// Window panel actions on tab switch
+
+vivaldi.jdhooks.hookModule('vivaldiSettings', (moduleInfo, exports) => {
+    let oldGetDefault = exports.getDefault
+    exports.getDefault = name => {
+        switch (name) {
+            case 'WINDOW_PANEL_TAB_SWITCHED_ACTIONS': return {
+                expandToActive: true,
+                selectActive: true
+            }
+            default: return oldGetDefault(name)
+        }
+    }
+    return exports
+})
+
+vivaldi.jdhooks.hookClass('tabs_WindowTree', cls => {
+    const PageStore = vivaldi.jdhooks.require('_PageStore')
+    return vivaldi.jdhooks.insertWatcher(class extends cls {
+        constructor(...e) {
+            super(...e)
+
+            let old_onPageStoreChanged = this._onPageStoreChanged
+            this._onPageStoreChanged = () => {
+                old_onPageStoreChanged()
+                const e = PageStore.a.getActivePage()
```

Comment:
Remove `.a`
I've rewritten `require` so it uses `exports` member of `moduleSignatures` (see jdhooks.js) which potentially allows specifying which of these `.a`, `.b` etc to select by meaningful name ("default" is default one :) ). I'll update wiki later.